lib/ttnt/testtask.rb

```diff
@@ -22,9 +30,19 @@ def initialize(rake_testtask)
 
       @anchor_description = 'Generate test-to-code mapping' + (@rake_testtask.name == :test ? '' : " for #{@rake_testtask.name}")
       @run_description = 'Run selected tests' + (@rake_testtask.name == :test ? '' : "for #{@rake_testtask.name}")
+
+      @@instance_by_name[@rake_testtask.name.to_s] = self
       define_tasks
     end
 
+    # Returns array of test file names.
+    #   Unlike Rake::TestTask#file_list, patterns are expanded.
+    def expanded_file_list
+      test_files = Rake::FileList[@rake_testtask.pattern].compact
+      test_files += @test_files.to_a if @test_files
```

Comment:
We need to regard both `pattern` and `test_files` options specified for `Rake::TestTask` since it is possible that both options are specified by a user. (`Rake::TestTask#file_list` is [doing the same](https://github.com/ruby/rake/blob/master/lib/rake/testtask.rb#L146-L149))

> you are missing a test where this line would be useful

That's right! :bulb: Thank you for your suggestion :+1: I added it here: https://github.com/Genki-S/ttnt/commit/2ca95e7ca074562de55dc1d03e96c4f828b09749